packages/react-components/react-list/stories/src/List/ListActiveElement.stories.tsx

```diff
@@ -56,16 +56,16 @@ export const ListActiveElement = () => {
 
   const [selectedItems, setSelectedItems] = React.useState<SelectionItemId[]>(['Melda Bevel']);
 
-  const onSelectionChange = React.useCallback((_, data) => {
+  const onSelectionChange = React.useCallback<NonNullable<ListProps['onSelectionChange']>>((_, data) => {
     setSelectedItems(data.selectedItems);
   }, []);
 
-  const onFocus = React.useCallback(event => {
+  const onFocus = React.useCallback((event: React.FocusEvent<HTMLLIElement>) => {
     // Ignore bubbled up events from the children
     if (event.target !== event.currentTarget) {
       return;
     }
-    setSelectedItems([event.target.dataset.value]);
+    setSelectedItems([event.currentTarget.dataset.value as SelectionItemId]);
```

Comment:
this is a runtime change, are you sure behaviours stay the same ?

Reply:
Yes, works as expected